Aquifer/IO.swift

```diff
@@ -12,13 +12,13 @@ import Foundation
 import Swiftz
 
 /// Returns a `Pipe` that reads input from `stdin` line-by-line and terminates on end-of-input.
-public func stdinLn<UO, UI>() -> Proxy<UO, UI, (), String, ()> {
+public func stdinLn() -> Producer<String, ()>.T {
```

Comment:
I'll leave it here and see if anything bad happens then.  I didn't change it the last time because the type signatures seemed... off.  But it kinda throws a wrench in the tutorial if that part is gone.